### PR TITLE
Add membership

### PR DESCRIPTION
**Pull request by BinaryHB0916**

### CODE REVIEW

File: community-membership.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
+# Community Membership [Draft]
+
+*This doc is in pre-release, it will be offcially released after 3 months from the date of its first commit*
+
+All contributors should be welcomed to the community by existing members, helped with PR workflow, and directed to relevant documentation and communication channels.
+
+This doc outlines the various responsibilities of contributor roles in
+KubeVela.
```
Comment (hongchaodeng) [inline community-membership.md:8]: merge with the above line

File: community-membership.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+| Role | Responsibilities | Requirements | Defined by |
+| -----| ---------------- | ------------ | -------|
+| Member | Active contributor in the community | have made multiple contributions to the project | OAM GitHub org member|
```
Comment (hongchaodeng) [inline community-membership.md:12]:
```suggestion
| Member | Active participation in the community | have made multiple contributions to the project | OAM GitHub org member|
```

File: community-membership.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+| Reviewer | Review contributions |  Active contributor and/or code reviewer. Responsible with major features | [OWNERS] file reviewer entry |
+| Approver | Contributions acceptance approval| Highly experienced active reviewer and contributor. On charge of certain product area| [OWNERS] file approver entry|
```
Comment (hongchaodeng) [inline community-membership.md:14]:
```suggestion
| Approver | Approve and commit contributions | Highly experienced active reviewer and contributor. On charge of certain project domain(s) | [OWNERS] file approver entry|
```

File: community-membership.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+
+All contributors should be welcomed to the community by existing members, helped with PR workflow, and directed to relevant documentation and communication channels.
+
+This doc outlines the various responsibilities of contributor roles in
+KubeVela.
+
+| Role | Responsibilities | Requirements | Defined by |
+| -----| ---------------- | ------------ | -------|
+| Member | Active contributor in the community | have made multiple contributions to the project | OAM GitHub org member|
+| Reviewer | Review contributions |  Active contributor and/or code reviewer. Responsible with major features | [OWNERS] file reviewer entry |
```
Comment (hongchaodeng) [inline community-membership.md:13]:
```suggestion
| Reviewer | Review contributions |  Active contributor and/or code reviewer. Responsible for major features | [OWNERS] file reviewer entry |
```

**Review by hongchaodeng — COMMENTED** [review]
lgtm

**Review by wonderflow — APPROVED** [review]
LGTM

**Review by leejanee — APPROVED** [review]

**Pull request merged**